### PR TITLE
Use #define for constants, avoid infinite loop in setup(), use const

### PR DESCRIPTION
**Pull request by ayan4m1**
This PR makes general code style changes aimed at increasing the readability of the C++ code.

### CODE REVIEW

File: gg_interface/src/main.cpp [hunk 1]
```diff
@@ -1,34 +1,32 @@
...
+        delay(10);
+    }
+    Serial.println("Gufu Gud");
 
     // Initilize the INA260
     if (!ina260.begin()) {
-        Serial.println("Couldn't find INA260 chip");
-        while (1);
+        Serial.println("Couldn't find INA260!");
+        return;
```
Comment (weeding-nerdy) [inline gg_interface/src/main.cpp]: Should we do any retries here instead of hard-failing after one attempt?

File: gg_interface/src/main.cpp [hunk 2]
```diff
@@ -39,21 +37,27 @@ void setup() {
     ina260.MaskEnable->read();
 
     // Enable alert on "Conversion Ready"
-    ina260.MaskEnable->write(0x0400);
+    ina260.MaskEnable->write(CONVERSION_READY_ADDRESS);
```
Comment (weeding-nerdy) [inline gg_interface/src/main.cpp]: Nitpick: 0x0400 is not a register address, it is an interrupt mask (or ALERT in the parlance of the INA260).

File: gg_interface/src/main.cpp [hunk 3]
```diff
@@ -39,21 +37,27 @@ void setup() {
...
-        ledState = false;
+        digitalWrite(LED_PIN, LOW);
     } else {
-        digitalWrite(ledPin, HIGH);
-        ledState = true;
+        digitalWrite(LED_PIN, HIGH);
     }
+
+    ledState = !ledState;
     delay(100);
```
Comment (weeding-nerdy) [inline gg_interface/src/main.cpp]: I had already changed this around to the following, it is simpler and eliminates the state variable:
```cpp
// Das blinkenlight!
digitalWrite(ledPin, !digitalRead(ledPin));
delay(500);
```

**Review by weeding-nerdy — CHANGES_REQUESTED** [review]
Thanks again for the code review! I made a few comments/nitpicks, but overall it looks great.

File: gg_interface/src/main.cpp (hunk 2 above)
Comment (ayan4m1) [reply gg_interface/src/main.cpp]: Good call, I'll update the var name!

File: gg_interface/src/main.cpp (hunk 3 above)
Comment (ayan4m1) [reply gg_interface/src/main.cpp]: Cool, I'll update this!

File: gg_interface/src/main.cpp (hunk 1 above)
Comment (ayan4m1) [reply gg_interface/src/main.cpp]: Good call!

**Review by weeding-nerdy — APPROVED** [review]
LGTM!

**Pull request merged**